docs/features/user-watchlists.md

```diff
@@ -63,8 +73,13 @@ To ensure a **cross-device experience**, we standardize on the following datafor
 
 ## Watchlist playlist
 
+<<<<<<< HEAD
```

Comment:
Looks like you've got a merge artifact here

Reply:
Thanks! Got some conflicts when merging updates from develop.